jupytercalpoly/Jupyter-Bifrost
# UI-updates

## Changes
- [x] Added icons
- [x] Move changing mark
- [x] Changed filter screen
- [x] Added adding a new pill screen
- [x] Only allow the scatter plot with point mark

## Testing
1. plot bifrost
2. click through the mark icons in edit tab to see the update to the main graph
3. Click filter (funnel) icon to see the update
4. Click plus icon to add a new pill.
    * Selecting a variable leads to the datafield tab
        * You can go back to variable selection by clicking the variable tab again. 
    * Selecting a datafield will create a new pill
